### ISSUE TITLE
support for multiple languages

### ISSUE THREAD

**Issue by chagara** [kozec/sc-controller#183]
is it possible to make this available in diff languages? at least the GUI

**Comment by kozec** [comment]
Yes, it is planned, but UI is changing too much right now. Keeping language files and all translations in sync while source strings are changing every day would get frustrating to everyone pretty fast.
